demo-app/build.gradle.kts

```diff
@@ -42,6 +45,16 @@ android {
     composeOptions {
         kotlinCompilerExtensionVersion = "1.5.13"
     }
+    val javaVersion = JavaVersion.VERSION_11
```

Comment:
Question: do we require (desugared) Java 11 features to use the android extension? For a demo app, it'd be nice to just use Kotlin and not having to worry about Java

Reply:
I think you're right -- at least right now it looks like the demo app is all kotlin, so I would think we could remove this and see if something else in the build requires it?